user-access-handlers/src/main/java/no/unit/nva/handlers/CreateUserHandler.java

```diff
@@ -232,7 +232,7 @@ public static ApiGatewayException castToCorrectRuntimeException(Failure<?> failu
         if (exception instanceof ConflictException) {
             return (ConflictException) exception;
         } else {
-            LOGGER.error(failure.getException().getMessage());
+            LOGGER.error("Failed to create or update user!", failure.getException());
```

Comment:
We want the whole stacktrace, not just the message?

Reply:
Or does it gets formatted by the logger?

Reply:
We want the source of the problem hidden in the stacktrace, yes. Currently it logs this:

ERROR - Expected a single value, but 0 were found